.github/workflows/trivy-scan.yml

```diff
@@ -12,11 +12,12 @@ jobs:
       - name: Get Latest Release Docker Image Sha
         id: latest-sha
         run: |
+          set -eo pipefail
           # Get the latest released docker image sha
           curl -sL https://api.github.com/repos/vmware-tanzu/carvel-kapp-controller/releases/latest | jq -r '.assets[] | select(.name | contains("release.yml")).browser_download_url' | wget -i -
 
-          echo ::set-output name=image::$(yq eval '.spec.template.spec.containers[0].image' release.yml -N -oj | jq 'select(. != null)' -r)
-          echo ::set-output name=tag::$(curl -sL https://api.github.com/repos/vmware-tanzu/carvel-kapp-controller/releases/latest | jq -r '.tag_name')
+          echo "{image}={$(yq eval '.spec.template.spec.containers[0].image' release.yml -N -oj | jq 'select(. != null)' -r)}"  >> $GITHUB_OUTPUT
+          echo "{tag}={$(curl -sL https://api.github.com/repos/vmware-tanzu/carvel-kapp-controller/releases/latest | jq -r '.tag_name')}" >> $GITHUB_OUTPUT
```

Comment:
Should we make [this](https://github.com/vmware-tanzu/carvel-kapp-controller/blob/c5f626e3f6a85d689621ae25f4edb0e57d8c3e55/.github/workflows/trivy-scan.yml#L142) change here as well?